dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_configurations_panel.scss

```diff
@@ -11,6 +11,33 @@
   border-right: $euiBorderThin;
   position: relative;
 
+  .euiConfigPanel {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+
+    .euiPanelFields {
+      @include euiYScrollWithShadows;
+      height: 90%;
+      overflow-y: scroll;
+    }
+
+    .euiFlexUpdateButton {
+      position: absolute;
```

Comment:
Do we really require absolute position inside a flex container?

Reply:
Yes it was not necessary So removed.